### ISSUE TITLE
Title Bar Buttons Not Highlighting fully

### ISSUE THREAD

**Issue by Zany-Zachary1** [B00merang-Project/Windows-10#201]
As I have known in Windows 10, the window buttons go all the way to the edge.  I have seen other themes do this, but yours doesn't.  Is there a way to fix this
![Screenshot from 2021-11-22 10-45-48](https://user-images.githubusercontent.com/61282566/142891700-2f0f6e5d-cf70-44e3-883c-7797442b2f03.png)
?

**Comment by Elbullazul** [comment]
You can remove headerbar padding, but that would also stretch all other headerbar buttons. If there wasn't a border around them it wouldn't be an issue. Maybe it's time I remove the border on the headerbar buttons?

**Comment by dominichayesferen** [comment]
Good luck making it hit the sides of the window in Firefox, I guess.

**Comment by Zany-Zachary1** [comment]
Yeah, Firefox is pretty buggy when it comes to theming.  They should have an option like chrome to use system title bar and borders.

**Comment by Elbullazul** [comment]
Note to self: The headerbar should look and behave a bit like Outlook in Windows 10:
![image](https://user-images.githubusercontent.com/15310985/224118605-0f50f3ad-2584-45fa-9eb6-7c9acaebe8c2.png)

the settings app looks similar:
![image](https://user-images.githubusercontent.com/15310985/224119051-9ea0c49e-141e-4249-b8e8-5ac3e2455e4d.png)

Improvements:
- flat buttons
- no titlebar spacing/padding

**Comment by dominichayesferen** [comment]
Firefox no longer supports being styled by GTK stylesheets.
